doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -462,6 +464,10 @@ Security libraries
 Modem libraries
 ---------------
 
+* :ref:`nrf_modem_lib_readme` :ref:`nrf_modem_lib_lte_net_if`:
+
+  * Fixed an issue with modem fault handling, where the event must be deferred from interrupt context before it can be forwarded to the zephyr :ref:`net_mgmt_interface` module.
```

Comment:
```suggestion
  * Fixed an issue with modem fault handling in the :ref:`nrf_modem_lib_lte_net_if`, where the event must be deferred from interrupt context before it can be forwarded to the Zephyr's :ref:`net_mgmt_interface` module.
```